operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+  * As you read the documentation, look out for any improvements that can be made \(outdated information, spelling/grammar, etc\)
 
 ### First Week
 
-Objective: Introduction to customers team members, resources, & make first contribution
+Objective: Introduction to customers team members, resources, and make first contribution
 
-Training Focus:
+Training focus:
 
 * Introduction to our Customer Obsession Principle
-* Introductions to team members through one on ones
-  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\) 
+* Introductions to team members through 1-1s
+  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\)
 * Introduction to product
-* Introduction to our “Fast, Obvious, Forgiving” design principles 
+* Introduction to our “Fast, Obvious, Forgiving” design principles
 * Introduction to key resources
 * Make your first contribution on GitHub
 
 Activities:
 
 Customer Obsession:
 
-* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week 
+* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week:
   * [CSM:PM Engagement](https://community-daily.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Emily, Rosie, Jenn\)
   * [Solution Architects](https://community-daily.mattermost.com/private-core/channels/solutions-architects)
   * [Pre-Sales Support](https://community-daily.mattermost.com/private-core/channels/pre-sales-support)
-* Join and review the topics and posts in these channels to get a better understanding of our customers 
+* Join and review the topics and posts in these channels to get a better understanding of our customers
   * [Customer Feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
   * [Customer Support](https://community-daily.mattermost.com/private-core/channels/community)
 * Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
 * Begin reading [docs.mattermost.com](https://docs.mattermost.com/)
 
 Team:
 
-* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings 
-  * Ask questions to understand development, design, QA processes 
-* Meet with Amy \(Release Manager\) and read the release process doc 
-* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\) - This will be scheduled for you 
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you
```

Comment:
This link is a 404. Confirm correct page is https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours.

Reply:
it should be - https://github.com/it33/readme/blob/master/README.md for Ian's readme file, and I *think* the "CEO Informational Meeting" linked in the readme is the first meeting people have (you can confirm with Amy Nichol if that's the right meeting).

Reply:
Thanks, @lfbrock. 

@amynicol1985, in the context of meeting with Ian for the first time is the link provided aimed at a specific meeting outline or just as an overview of Ian ahead of time?

Reply:
Will follow this up post-merge as it's not a blocker.